### ISSUE TITLE
Add resource usage summary page

### ISSUE THREAD

**Issue by tommyalatalo** [jippi/hashi-ui#521]
It would be incredibly useful to have a page that contains resource allocation and usage summaries, both in text form and as graphs.

What would be of interest to see is a page which contains separate (bar?) graphs for each client node, showing free/used/allocated cpu, memory and network resources in a compact way.

The same need is applicable on allocation basis as well, a page where all allocations would be listed and the same type of graphs would show all the allocations' resource usage on the same page.

In short there is a lack of overview of resource usage information, currently you have to go into each client node separately to check resource usage, and the same goes for allocations. Not entirely sure of where this new summary page should be placed, but a first idea is to simply add a new link in the main sidebar menu called "Resources"?
